src/main/java/dukeApp/duke/Duke.java

```diff
@@ -0,0 +1,48 @@
+package dukeApp.duke;
+import dukeApp.files.*;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+public class Duke {
```

Comment:
Hi Vanessa, I would recommend that you read the JavaDoc way of documenting your code. I guess having good documentation does help with other's understanding of the code!